### PR TITLE
[WSTEAM1-86] Switch recommendations endpoint

### PR DESCRIPTION
**Pull request by mfonofm**
Resolves #[WSTEAM1-86](https://jira.dev.bbc.co.uk/browse/WSTEAM1-86)

**Overall change:**
Switching from a public to a private recommendations endpoint

**Code changes:**

- Update `getRecommendationsUrl` with new private endpoint
- Remove `variant` from `getRecommendationsUrl`
- Update tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/StoryPage/index.test.jsx [hunk 1]
```diff
@@ -250,6 +250,7 @@ describe('Story Page', () => {
   const appEnv = process.env.SIMORGH_APP_ENV;
   beforeEach(() => {
     process.env.SIMORGH_ICHEF_BASE_URL = 'https://ichef.test.bbci.co.uk';
+    process.env.RECOMMENDATIONS_ENDPOINT = 'http://mock-recommendations-path';
```
Comment (andrewscfc) [inline src/app/pages/StoryPage/index.test.jsx:253]: To ensure you don't cause side-effects on other tests I think you need a `delete process.env.RECOMMENDATIONS_ENDPOINT;` in an `afterEach`. `process.env` vars are not automatically cleaned up between tests.

Incidently I'd check if `process.env.SIMORGH_ICHEF_BASE_URL` has an afterEach as well as that will have the same issue.

**Review by andrewscfc — COMMENTED** [review]
All looks great aside from the test cleanup 👍

File: src/app/pages/StoryPage/index.test.jsx (hunk 1 above)
Comment (mfonofm) [reply src/app/pages/StoryPage/index.test.jsx:253]: Good spot

**Review by andrewscfc — APPROVED** [review]
LGTM thanks for sorting the test

**Review by RetiredBlob — APPROVED** [review]

**Pull request merged**